src/sdk/accountspace_fs.js

```diff
@@ -678,8 +716,8 @@ class AccountSpaceFS {
         }
     }
 
-    async _check_username_already_exists(action, username) {
-        const account_config_path = this._get_account_config_path(username);
+    async _check_username_already_exists(action, username, root_name) {
```

Comment:
What will happen if a root accounts manager tries to create a root account?
Would you please add comments in the function?

Reply:
This logic is handled in _get_account_config_path_by_requesting_account(), before this function.
(I've added some doc to that function).